src/app/shared/issue-tables/issue-tables.component.html

```diff
@@ -145,6 +149,27 @@
     </mat-cell>
   </ng-container>
 
+  <!--Tester Disagree Column-->
+  <ng-container matColumnDef="testerDisagree">
+    <mat-header-cell mat-header-cell *matHeaderCellDef mat-sort-header>Response</mat-header-cell>
+    <mat-cell *matCellDef="let issue">
+      <span
+        (click)="$event.stopPropagation()"
+        *ngIf="issue.testerDisagree"
+        [ngStyle]="this.labelService.setLabelStyle(this.labelService.getColorOfLabel('Rejected'))"
+      >
+        Disagree
```

Comment:
Given the terminology in the codebase and just from how it sounds, I think using "Accepted" and "Rejected" would be better here.

Reply:
Since the way the tester response is done is different from team response, i.e. tester response is a checkbox of `I agree` and `I disagree`, vs team response as selecting from a dropdown of github labels, I was hesitant to use the same terminology between the two. Would it still be better to have the terminology changed to Accepted/Rejected in the code (and maybe display Accepted/ Rejected instead of Agree/Disagree as well)?

Reply:
Since the way the tester response is done is different from team response, i.e. tester response is a checkbox of `I agree` and `I disagree`, vs team response as selecting from a dropdown of github labels, I was hesitant to use the same terminology between the two. Would it still be better to have the terminology changed to Accepted/Rejected in the code (and maybe display Accepted/ Rejected instead of Agree/Disagree as well @damithc )?  @dingyuchen

Reply:
From the user POV, devs are in a position to accept/reject (it's _their_ project, and they are _receivers_ of bug reports) bug reports but testers can only agree/disagree with the dev decision. Hence, we use two different set of terms.
OK to use same terms internally but perhaps it's better for internal terms to follow external terms or else it can become another source of confusion.

Reply:
Ah I see, then Agree/Disagree would make more sense here, no need to make changes @gycgabriel